_data/contributors.yml

```diff
@@ -986,17 +986,16 @@
   github_username: sneakers-the-rat
   website: https://jon-e.net
   github_image_id: 12961499
-  mastodon:
   twitter: json_dirs
-  bio: Bad at programming in beautiful Oregon
-  orcidid:
+  mastodon: https://neuromatch.social/@jonny
+  orcidid: 0000-0003-0545-5066
+  bio: Information liberation is for the children <3
   contributor_type:
     - guidebook-contrib
+  email: j@nny.fyi
   packages-submitted:
   packages-reviewed:
-  location:
-  organization:
-  email:
```

Comment:
@sneakers-the-rat we can keep these empty but essentiall i think the script might fail if we delete them. i parse everyone's github profile to grab information that is publish. the bio comes from your bio. so i'm ok with it being empty but deleting might cause issues.